lensless/benchmark.py

```diff
@@ -251,7 +251,12 @@ def benchmark(model, dataset, batchsize=1, metrics=None, **kwargs):
         metrics = {
             "MSE": MSELoss().to(device),
             "MAE": L1Loss().to(device),
-            "LPIPS": lpip.LearnedPerceptualImagePatchSimilarity(net_type="vgg").to(device),
+            "LPIPS_V": lpip.LearnedPerceptualImagePatchSimilarity(
+                net_type="vgg", normalize=True
+            ).to(device),
+            "LPIPS_A": lpip.LearnedPerceptualImagePatchSimilarity(
```

Comment:
how about "LPIPS_VGG" and "LPIPS_Alex"? I think we can afford a few more characters ;)